pkgs/by-name/wi/wifi-qr/package.nix

```
@@ -0,0 +1,85 @@
+{ lib
+, fetchFromGitHub
+, installShellFiles
+, makeWrapper
+, gnome
+, ncurses
+, networkmanager
+, patsh
+, qrencode
+, stdenvNoCC
+, xdg-utils
+, zbar
+}:
+stdenvNoCC.mkDerivation {
+  pname = "wifi-qr";
+  version = "unstable-2023-09-30";
+
+  outputs = [ "out" "man" ];
+
+  src = fetchFromGitHub {
+    owner = "kokoye2007";
+    repo = "wifi-qr";
+    rev = "821892001f735dc250a549ea36329cdc767db9c9";
+    hash = "sha256-kv0qjO+wn4t//NmKkHB+tZB4eRNm+WRUa5rij+7Syuk=";
+  };
+
+  buildInputs = [
+    gnome.zenity
+    ncurses
+    networkmanager
+    qrencode
+    xdg-utils
+    zbar
+  ];
+
+  nativeBuildInputs = [
+    installShellFiles
+    makeWrapper
+    patsh
+  ];
+
+  dontBuild = true;
+
+  dontConfigure = true;
+
+  postPatch = ''
+    substituteInPlace src/wifi-qr.desktop \
+      --replace "Exec=sh -c 'wifi-qr -g'" "Exec=$out/bin/wifi-qr -g" \
+      --replace "Exec=sh -c 'wifi-qr -q'" "Exec=$out/bin/wifi-qr -q" \
+      --replace "Exec=sh -c 'wifi-qr -p'" "Exec=$out/bin/wifi-qr -p" \
+      --replace "Exec=sh -c 'wifi-qr -c'" "Exec=$out/bin/wifi-qr -c" \
+      --replace "Icon=wifi-qr.svg" "Icon=wifi-qr"
```

Maybe these cases could be merged?

```suggestion
    sed -E -i src/wifi-qr.desktop \
      -e "s/Exec=sh -c '(.*)'/Exec=\1/" \
      -e "s/.svg//"
```